README.md

```diff
@@ -130,16 +130,17 @@ class BookStore extends ImmutableStore<Book[]> {
           })),
         ],
     });
-
-    // Handle store reset request events
+    
+    // Handle store reset request events. Note, the storeResetRequestEvent would 
+    // be created or imported, see the events documentation for more details
```

Comment:
Let me know if you'd like more context here. For someone reading the docs, I was confused because `storeResetRequestEvent` looked like it was an object/variable but it was not being imported or created in the docs. So I added an "FYI" this is actually an object and a note to refer to the events page